x-pack/filebeat/input/awscloudwatch/config.go

```diff
@@ -0,0 +1,57 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package awscloudwatch
+
+import (
+	"errors"
+	"time"
+
+	"github.com/elastic/beats/v7/filebeat/harvester"
+	awscommon "github.com/elastic/beats/v7/x-pack/libbeat/common/aws"
+)
+
+type config struct {
+	harvester.ForwarderConfig `config:",inline"`
+	LogGroupARN               string              `config:"log_group_arn"`
+	LogGroupName              string              `config:"log_group_name"`
+	RegionName                string              `config:"region_name"`
+	LogStreams                []string            `config:"log_streams"`
+	LogStreamPrefix           string              `config:"log_stream_prefix"`
+	StartPosition             string              `config:"start_position" default:"beginning"`
+	ScanFrequency             time.Duration       `config:"scan_frequency" validate:"min=0,nonzero"`
+	APITimeout                time.Duration       `config:"api_timeout" validate:"min=0,nonzero"`
+	APISleep                  time.Duration       `config:"api_sleep" validate:"min=0,nonzero"`
+	AwsConfig                 awscommon.ConfigAWS `config:",inline"`
+}
+
+func defaultConfig() config {
+	return config{
+		ForwarderConfig: harvester.ForwarderConfig{
+			Type: "awscloudwatch",
+		},
+		StartPosition: "beginning",
+		ScanFrequency: 1 * time.Minute,
```

Comment:
This may be too high? I guess 10-20s will get you to data faster without hitting too much the API

Reply:
Good point, I will change the default to 10 seconds.